Compos/1_Beam/WebOpening/WebOpening.cs

```diff
@@ -235,14 +241,14 @@ public override string ToString() {
         case OpeningType.Start_notch:
         case OpeningType.End_notch:
         case OpeningType.Rectangular:
-          if (Width == null || Height == null) {
+          if (ComposUnitsHelper.IsEqual(Width, Length.Zero) || ComposUnitsHelper.IsEqual(Height, Length.Zero)) {

```

Comment:
you lost null check. Comparing to zero will throw an error which is not handled

Reply:
Width and Height, can never be null, they are of non nullable type

Reply:
@psarras In C# a struct is a 'value type', which can't be null 
![image](https://github.com/user-attachments/assets/e0548040-1bcd-476b-9b9d-34e045dbbc07)